WordPress/Classes/ViewRelated/Views/List/NotificationsList/AvatarView.swift

```diff
@@ -3,7 +3,7 @@ import Gravatar
 import DesignSystem
 import WordPressUI
 
-struct AvatarsView<S: Shape>: View {
+struct AvatarView<S: Shape>: View {
```

Comment:
I ended up not using it for this PR to avoid adding too many changes, but I was always so annoyed by the naming, I decided to change it 😅